src/spotify/utils.ts

```diff
@@ -10,7 +10,7 @@ export function createCancelablePromise<T>(
         cancel = () => {
             reject(CANCELED_REASON);
         };
-        executor(resolve, reject);
+        executor(resolve as any, reject);
```

Comment:
With line 5's code changed to
```typescript
    executor: (resolve: (value: T | PromiseLike<T>) => void | undefined,
```

this could just be
```suggestion
executor(resolve, reject);
```